packages/lambdas/src/sqs-to-converter.ts

```diff
@@ -256,48 +287,60 @@ export const handler = Sentry.AWSLambda.wrapHandler(async (event: SQSEvent) => {
-        //eslint-disable-next-line @typescript-eslint/no-explicit-any
-      } catch (err: any) {
+      } catch (error) {
         // If it timed-out let's just reenqueue for future processing - NOTE: the destination MUST be idempotent!
         const count = message.attributes?.ApproximateReceiveCount
           ? Number(message.attributes?.ApproximateReceiveCount)
           : undefined;
         const isWithinRetryRange = count == null || count <= maxTimeoutRetries;
-        const isRetryError = isAxiosTimeout(err) || isAxiosBadGateway(err);
+        const isRetryError = axios.isAxiosError(error)
+          ? isAxiosTimeout(error) || isAxiosBadGateway(error)
+          : false;
+        const networkErrorDetails = getNetworkErrorDetails(error);
+        const { details, code, status } = networkErrorDetails;
         if (isRetryError && isWithinRetryRange) {
-          const details = `${err.code}/${err.response?.status}`;
           console.log(
-            `Timed out (${details}), reenqueue (${count} of ${maxTimeoutRetries}): `,
-            message
+            `Timed out (${code}/${status}), reenqueue (${count} of ` +
+              `${maxTimeoutRetries}), lambdaParams ${lambdaParams}`
           );
-          capture.message("Conversion timed out", {
-            extra: { message, context: lambdaName, retryCount: count },
+          capture.message("Conversion timed out - retrying", {
+            extra: { message, ...lambdaParams, context: lambdaName, retryCount: count },
+            level: "info",
           });
           await sqsUtils.reEnqueue(message);
         } else {
-          console.log(`Error processing message: ${JSON.stringify(message)}; \n${err}: ${err}`);
-          // Axios error response
-          if (err.response) {
-            const resp = err.response;
-            const responseData = resp.data ? JSON.stringify(resp.data) : "undefined";
+          const msg = "Error processing message on " + lambdaName;
+          console.log(
+            `${msg} - lambdaParams: ${lambdaParams} - ` +
+              `error: ${JSON.stringify(networkErrorDetails)}`
+          );
+          if (axios.isAxiosError(error)) {
+            const responseData = error.response?.data
+              ? JSON.stringify(error.response.data)
+              : "undefined";
             console.log(`Response body: ${responseData}`);
           }
-          capture.error(err, {
-            extra: { message, context: lambdaName, retryCount: count },
+          capture.error(msg, {
+            extra: { message, ...lambdaParams, context: lambdaName, networkErrorDetails, error },
           });
           await sqsUtils.sendToDLQ(message);
 
-          await ossApi.notifyApi({ cxId, patientId, status: "failed", jobId, source }, log);
+          // TODO not sure if should retry on timeout
+          await executeWithNetworkRetries(
+            () => ossApi.notifyApi({ ...lambdaParams, status: "failed", details }, log),
```

Comment:
wrap in oss-api.ts instead of here

Reply:
Good call, also updated. 👍

Reply:
Ouch, I missed this, thought it was another one. Done now, thanks!